lib/model/products.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+
+class product{
+   IconData favorite;
+   String imageUrl;
+   String productTitle;
+   String productSubtitle;
+
+    product({required this.favorite,
+    required this.imageUrl,
+    required this.productTitle,
+    required this.productSubtitle});
+}
+
+
```

Comment:
make a function <<<< List<Product> getProductList() >>>> and call this where u need the list of products